### PR TITLE
[WIP] Linkifying User Inputted Text 

### PR DESCRIPTION
**Pull request by codybousc**
I'm labelling this as WIP because it doesn't catch edge cases. It works for every instance that the full url is given, but doesn't catch cases like: 

bit.ly/citizensengage

One potential solution may be using regex for edge cases?

### CODE REVIEW

**Comment by toolness** [comment]
Oh, nice detective work regarding [`urlize()`](https://docs.djangoproject.com/es/1.9/ref/templates/builtins/#urlize), I didn't even know about that!

I think it's OK for there to be some edge cases (in fact, it looks like that bit.ly URL doesn't get noticed by GitHub either!). The hard thing about solving this problem is that doing it ever so slightly the wrong way could result in a [cross-site scripting (XSS)](https://en.wikipedia.org/wiki/Cross-site_scripting) vulnerability, so I prefer to use peer-reviewed third-party code for this one. Granted, we've got insurance against XSS through NoI's built-in support for Content Security Policy (#70), but better safe than sorry!

Thanks also for adding a test!

File: app/tests/test_views.py [hunk 1]
```diff
@@ -234,6 +234,18 @@ def test_step_3_with_invalid_areaid_is_not_found(self):
         self.assert404(self.client.get('/register/step/3/blah/1'))
 
 class ActivityFeedTests(ViewTestCase):
+    def test_activity_link_present(self):
+        self.login()
+        user = User.query_in_deployment()\
+                 .filter(User.email == 'test@example.org').one()
+        user.first_name = 'John'
+        user.last_name = 'Doe'
```
Comment (toolness) [inline app/tests/test_views.py]: Is there a particular reason you're doing this stuff with the `user` object here?

**Comment by codybousc** [comment]
I had to look to the other tests to figure out how to write it and I took much of the body of 'test_posting_activity_works' on line 269. You're right though, now that I look at it, the user object isn't needed and the test passes without it. I'll update the request!

**Comment by codybousc** [comment]
I'm not sure if you all have a certain protocol for closing out PRs, so I'll just say that the code for 'linkifying' user's inputted text has been reviewed and passes unit testing.

**Pull request closed without merging**